warframe.js

```diff
@@ -1,323 +1,138 @@
-/**
- * Warframe.js by Luca Kiebel
- *
- * Docs and more: https://github.com/lucakiebel/Warframe.js
- */
+'use strict';
 
-const request = require('request-promise');
+const fetch = require('node-fetch');
+
+const BASE = 'https://api.warframestat.us';
```

Comment:
my only worry for web usage is this :point_up: I don't really like making extraneous globals, but i couldn't find a good way without making more class stuff... i might try it as a pseudo-static var so we don't expose excessive amounts of vars

Reply:
I moved it in, but I'd love feedback. I wanted to make it a static const, but that'd require doing babel and stuff